setup.py

```diff
@@ -100,24 +137,34 @@ class DevelopMode(develop):
     def run(self):
         """Install the package in a developer mode."""
         super().run()
-        for _file in ETC_FILES:
-            self.create_path(_file)
 
-    @staticmethod
-    def create_path(file_name):
-        """Method used to create the configurations files using develop."""
-        etc_kytos_dir = os.path.join(BASE_ENV, 'etc/kytos')
+        self.create_paths()
+        self.generate_file_from_template(TEMPLATE_FILES, prefix=BASE_ENV)
+
+        for file_name in ETC_FILES:
+            self.generate_file_link(file_name)
 
+        for file_name in TEMPLATE_FILES:
+            self.generate_file_link(file_name.replace('.template', ''))
+
+    @staticmethod
+    def generate_file_link(file_name):
+        """Method used to create a symbolic link from a file name."""
         current_directory = os.path.dirname(__file__)
         src = os.path.join(os.path.abspath(current_directory), file_name)
         dst = os.path.join(BASE_ENV, file_name)
 
-        if not os.path.exists(etc_kytos_dir):
-            os.makedirs(etc_kytos_dir)
-
         if not os.path.exists(dst):
             os.symlink(src, dst)
 
+    @staticmethod
+    def create_paths():
+        """Method used to create the paths used by Kytos in develop mode."""
+        directories = [os.path.join(BASE_ENV, 'etc/kytos')]
+        for directory in directories:
+            if not os.path.exists(directory):
+                os.makedirs(directory)
+
 
 requirements = [i.strip() for i in open("requirements.txt").readlines()]
 napps_dir = os.path.join(BASE_ENV, 'var/lib/kytos/napps/.installed')
```

Comment:
I understand your point.

But we are creating this directory at this point because something would break later on while using the project.

Now consider that we have created this directory during the setup, and then the user changes it on the generated `kytos.conf` file (for example to `/tmp/napps_dir/`). Then, something on kytos or kytos-utils will be broken, since the folder does not exists.

So, what I am saying is that every "code" that reads from the `kytos.conf` and may use this directory should check if it exist prior to using it.

(By the way, this has made me think about the relationship between `kytos.conf` [kytos project] and `~/.kytosrc`, [kytos-utils project]. Shouldn't they be aligned somehow?)